src/ucs/debug/debug.c

```diff
@@ -156,6 +156,8 @@ static int dl_match_address(struct dl_phdr_info *info, size_t size, void *data)
     return 0;
 }
 
+#ifdef HAVE_DETAILED_BACKTRACE
```

Comment:
move `#ifdef HAVE_DETAILED_BACKTRACE` before `dl_match_address` function declaration to fix compiler warning from testing?